src/main/java/hudson/plugins/promoted_builds/conditions/ManualCondition.java

```diff
@@ -95,18 +98,28 @@ public ParameterDefinition getParameterDefinition(String name) {
         return null;
     }
 
-    public Set<String> getUsersAsSet() {
+    public Set<String> getUsersAsSet(AbstractBuild<?,?> build) {
```

Comment:
Breaks binary compatibility, although I don't see any other callers based on a GitHub search of the jenkinsci organization, so it might not mater. We could always be safe and keep the previous method around as well, and have it call the new method passing `null` for the new parameter.

Same issue with `isInUsersList` and `isInGroupList `.